assets/js/state/sagas/hosts.js

```diff
@@ -12,6 +54,20 @@ export function* hostDeregistered({ payload }) {
   );
 }
 
+function* watchHostDeregisterableDetail() {
+  while (true) {
```

Comment:
I have to use this pattern instead of [takeEvery()](https://redux-saga.js.org/docs/api/#takeeverypattern-saga-args) in order to be able to pass `debounce` into the action's `payload`.

https://redux-saga.js.org/docs/api/#notes-3